outbound/queue.js

```diff
@@ -48,6 +48,7 @@ let queue_count = 0;
 
 exports.get_stats = function () {
     return in_progress + '/' + exports.delivery_queue.length() + '/' + exports.temp_fail_queue.length();
+    // return ${in_progress}/${exports.delivery_queue.length()}/${exports.temp_fail_queue.length()}`;
```

Comment:
why is this commented out?

Reply:
My focus is towards log messages, but i thought, we can apply template literals in other places as well like in this return statement. 

If it's according to your coding style you may use it, if not please ignore.

Reply:
But you didn't apply template literals. That would have been fine. Instead, you commented it out.